service/repository/test_file_repository.py

```
@@ -0,0 +1,157 @@
+        mock_repository = FileRepository(file_path, MockModel)
+        mock_object = self.__prepare_mock_object()
+
+        # when
+        try:
+            FileRepository.post(mock_repository, mock_object)
+            FileRepository.post(mock_repository, mock_object)
+            FileRepository.post(mock_repository, mock_object)
+            FileRepository.post(mock_repository, mock_object)
+            FileRepository.post(mock_repository, mock_object)
+
+            with open(file_path, "r") as file:
```

Ruff

Unnecessary open mode parameters

Remove open mode parameters

(UP015)